Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># EventsApp\n<h1>App</h1>\n<h3>working!!!</h3>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 077043483326b627ec7f7a61db644c7adfd62cec)
+++ b/README.md	(date 1658145409532)
@@ -1,3 +1,3 @@
 # EventsApp
-<h1>App</h1>
+<h1>APP</h1>
 <h3>working!!!</h3>
